tests/lib/rules/indent.js

```diff
@@ -185,7 +202,7 @@ ruleTester.run("indent", rule, {
         {
             code:
             "require('http').request({hostname: 'localhost',\n" +
-            "                         port: 80}, function(res) {\n" +
+            "  port: 80}, function(res) {\n" +
```

Comment:
This change would undo whatever was done for https://github.com/eslint/eslint/issues/4075, is that ok?